docs/Running-Tachyon-with-RDMA.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: global
+title: Running Tachyon with RDMA over JXIO
+---
+
+## Prerequisite
+
+* **Compile JXIO**. Download and compile [JXIO](https://github.com/accelio/JXIO/).
+Tachyon is supported by JXIO v1.3 and above
```

Comment:
yes, "git checkout v1.3", it's the latest GA.
master branch can be used as well
versions prior to 1.3 will have compatibility issues